docs/source/users/problem_definition_files/spinw.rst

```diff
@@ -15,105 +32,132 @@ software, name, description
   As described in the native format.
 
 input_file
-  For SpinW we require an SQW file containing the data from Horace.
-
-projection
-  The projection defines the axis that the data is projected onto for the fit.
-
-  This follows a similar pattern to the function in that it is a comma
-  seperated list of key-value pairs and has several parameters.
-  Usually only u, v, and type are used.
-  - u, v, and w: 3D vectors used to define the viewing coordinate system.
-                 w will be defined as normal to the plane spanned by u and v
-                 if not given.
-  - type: 3 chars (a, r, or p) to define the type of axes.
-          a - The axis is measured in angstroms,
-          r - The axis is measured in rlu (scaled so the 1-norm == 1),
-          p - The axis is not altered (preserve values)
-  - uoffset: 4D vector, if the data needs recentreing this can be used
-  
-  Vectors can be defined as comma seperated values in square brackets
-  `[val1, val2, ...]`
-
-  e.g.::
-    projection = 'u=[1,0,0],v=[0,1,0],type=rrr'
-
-cut
-  The cut defines the extent of the data and consists of 4 ranges for binning
-  boundaries along u, v, w, and energy axes defined in projection.
-  Each range can be defined by:
-  - no value: Keep the data as stored in the file
-  - single value: Sets a bin width, the data is rebinned into this width
-  - 3 values => lower bound, bin width, upper bound. The data is truncated and rebinned.
-  - 2 values => lower and upper bound. The data is truncated into a single bin.
-
-  e.g.::
-    cut = 'p1_bin=[-1,0.05,1],p2_bin=[-1,0.05,1],p3_bin=[-10,10],p4_bin=[10,20]'
-
-sample
-  The sample defines the target of the experiment and has several parameters.
-
-  This must have a class as "IX_Sample", the remaining arguments are passed
-  through to the IX_Sample class constructor.
-  Full details can be found `here <https://pace-neutrons.github.io/Horace/v3.6.2/user_guide/Resolution_convolution.html#the-tobyfit-class>`__.
+  For SpinW we require an SQW file or a MAT file containing the data from Horace. 
   
-  e.g.::
-    sample = 'class=IX_sample,xgeom=[0,0,1],ygeom=[0,1,0],shape='cuboid',ps=[0.01,0.05,0.01])
+.. note::
+  The MAT file should be used if you are loading in multiple objects. Make sure 
+  to load the data in appropriately in the `wye_function`. 
+
+function
+  The function is defined by one or two matlab files which returns a SpinW model of the foreground or the foreground and background
+  repectively separated by a semicolon.
+
+  The format is again comma seperated key-value pairs, with the matlab files 
+  defined by the variable "foreground" and "background" and the remaining pairs defining starting
+  values as in the native parser.
```

Comment:
The format of what? Parameters?
> Parameters should be specified as a comma-separated list of ``key=value`` pairs.

Reply:
This is for the functions and the parameters. e.g.
`foreground=funcF(x1,x2 ,x3), x1=1, x2=2, x3=3  `    **;**         `background=funcB(x4,x5,x6), x4=4, x5=5, x5=5  `